flytekit/core/base_task.py

```diff
@@ -286,16 +294,37 @@ def __call__(self, *args, **kwargs):
                     return create_task_output(vals, self.python_interface)
             return self._local_execute(ctx, **kwargs)
         else:
-            logger.warning("task run without context - executing raw function")
-            new_user_params = self.pre_execute(ctx.user_space_params)
+            logger.info("Task run without context - executing raw function")
             with FlyteContextManager.with_context(
                 ctx.with_execution_state(
-                    ctx.execution_state.with_params(
-                        mode=ExecutionState.Mode.LOCAL_TASK_EXECUTION, user_space_params=new_user_params
-                    )
+                    ctx.new_execution_state().with_params(mode=ExecutionState.Mode.LOCAL_WORKFLOW_EXECUTION)
                 )
-            ):
-                return self.execute(**kwargs)
+            ) as child_ctx:
+                result = self._local_execute(child_ctx, **kwargs)
```

Comment:
update  the comment in local execute. It is currently
```
        This code is used only in the case when we want to dispatch_execute with outputs from a previous node
        For regular execution, dispatch_execute is invoked directly.
```

I think it should be should be used when we have native values and want to execute?